prow/config/config.yaml

```diff
@@ -368,6 +374,7 @@ presubmits:
   - name: gomod
     branches:
     - main
+    - release-1.3
```

Comment:
Why do we add it here? Or maybe better question, why wasn't release-1.2 here before?

Reply:
release-1.2 is one block down, its in separate block because of golang docker image version

Reply:
Ah that explains it! Thanks!